src/logging.cc

```diff
@@ -2568,6 +2568,10 @@ void InitGoogleLogging(const char* argv0,
 }
 #endif
 
+bool IsGoogleLoggingInitialized() {
+  return glog_internal_namespace_::IsGoogleLoggingInitialized();
```

Comment:
The internal definition can be moved directly to the outer namespace. No aliasing necessary.

Reply:
There're still possible ways to touch that symbol, e.g. when linking glog as static lib.
I don't want to break that backward compatibility unnecessarily.

Reply:
Backwards compatibility for internal symbols does not matter,

Reply:
I would say this one is a little bit special.
Because of its high importance (many use cases doesn't work without it), I believe there're many code out there touching this symbol regardless.
If we can easily keep the compatibility for free, why do we need to break it for nothing?

Let me know if I missed any strong benefit we can get by breaking it.

Reply:
The benefit is clean code. A change to an internal symbol does not justify maintaining compatibility that was not guaranteed in the first place.

Reply:
> The benefit is clean code.

I can make the change if you think that's better.

> Your argument does not justify maintaining compatibility that was not guaranteed in the first place.

Exactly and intended.
IMHO when there's maintenance needed, it can be simply removed at that time as the cost comes.
Before that happens, there's be a free gap for graceful deprecation.

> In particular, the "breaking change" to the internal symbol has already occurred ...

Not for static build.
The change is only on visibility of shared libs.

Reply:
Emmm, if I move it to logging.cc, I'll also need to expose the locally defined `g_program_invocation_short_name` itself probably into a header, or introduce another function to hide it that does basically the same as internal `IsGoogleLoggingInitialized`. Don't think that's a good idea.

Reply:
Please make the change.

Reply:
Changed to exposing `g_program_invocation_short_name`.